remoulade/api/main.py

```diff
@@ -47,6 +67,14 @@ def get_scheduled_jobs():
     return {"result": [job.as_dict() for job in scheduled_jobs.values()]}
 
 
+@app.route("/messages", methods=["POST"])
+def enqueue_message():
```

Comment:
add an error handler for marshmallow validation returning a error 400 with the content of the error message